dbScript/buildDB.js

```diff
@@ -1,14 +1,121 @@
 // exercise library json file import
 import * as exercise_library from "../exercise_library/exercise_library.json" assert { type: "json" }
+import { addDoc, collection, getFirestore } from "firebase/firestore";
+import { initializeApp } from "firebase/app";
 
-import db from "../myFitnessTrainer/firebaseConfig.js"
+// import { db } from "../myFitnessTrainer/firebaseConfig.js";
+
+// firebase config
```

Comment:
We can add a TODO here, similar to the one listed at https://github.com/umjennifer/myFitnessTrainer/blob/ab512050665c50e02816c6f1d91c17585031f5b5/myFitnessTrainer/firebaseConfig.js#L13 

I think moving forward, all our TODOs should include a link to an associated issue (ex: in GitHub Issue) so that we can track them all more easily

Reply:
Or task in Asana